modular_skyrat/modules/customization/modules/surgery/robot_chassis_restoration.dm

```diff
@@ -8,36 +8,48 @@
 	/datum/surgery_step/prepare_electronics,
 	/datum/surgery_step/add_plating/fullbody,
 	/datum/surgery_step/weld_plating/fullbody,
-	/datum/surgery_step/finalize_chassis_restoration)
+	/datum/surgery_step/finalize_positronic_restoration)
 
 	target_mobtypes = list(/mob/living/carbon/human)
 	possible_locs = list(BODY_ZONE_CHEST)
 	requires_bodypart_type = BODYPART_ROBOTIC
-	desc = "A surgical procedure that rebuilds a synthetic unit from their skeleton to full integrity, recommended if they are damaged far beyond repair."
+	desc = "A surgical procedure that reboots a positronic brain."
+
+/datum/surgery/robot_chassis_restoration/can_start(mob/user, mob/living/carbon/target)
+	if(!..())
+		return FALSE
+	if(target.stat != DEAD)
+		return FALSE
+	if(target.suiciding)
+		return FALSE
+	var/obj/item/organ/brain/target_brain = target.getorganslot(ORGAN_SLOT_BRAIN)
+	if(!target_brain)
+		return FALSE
+	return TRUE
 
 /datum/surgery_step/pry_off_plating/fullbody
 	time = 120
 
 /datum/surgery_step/pry_off_plating/fullbody/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
-	display_results(user, target, span_notice("You begin to pry off [target]'s plating..."),
-			"[user] begins to pry off [target]'s plating.",
-			"[user] begins to pry off [target]'s plating.")
+	display_results(user, target, span_notice("You begin to pry open compromised panels on [target]'s braincase..."),
```

Comment:
Why is there so many arguments and why is there only one of them that's in a `span_notice()`?

Reply:
The same question applies to all of the following ones. I ask because the other calls to it don't repeat the same string three times.

Reply:
Fixed this.